plugins/faustwp/includes/telemetry/functions.php

```diff
@@ -81,3 +94,21 @@ function get_plugin_version() {
 
 	return $version;
 }
+
+/**
+ * Returns the WPGraphql Content Blocks plugin version.
+ *
+ * @return null|string
+ */
+function get_wpgraphql_content_blocks_plugin_version() {
+	$plugin_file = 'wp-graphql-content-blocks/wp-graphql-content-blocks.php';
```

Comment:
Since it's possible to install this plugin in a different folder name, it might be better to derive the info a slightly different way.

`wp-graphql-content-blocks` defines a version constant that would simplify the logic a bit.

Something similar to this should work:

```
return defined( 'WPGRAPHQL_CONTENT_BLOCKS_VERSION' ) ? WPGRAPHQL_CONTENT_BLOCKS_VERSION : null;
```